README.rst

```diff
@@ -54,6 +62,7 @@ The following table describes the syntax conventions; the syntax for running the
 Required arguments:
 
 ``batch_scoring --host=<host> --user=<user> <project_id> <model_id> <dataset_filepath> --datarobot_key=<datarobot_key> {--password=<pwd> | --api_token=<api_token>}``
+``batch_scoring_sse --host=<host> <import_id> <dataset_filepath>``
```

Comment:
I am not sure how this actually displays in the readme, but from the "view" option here, this command is part of the one above. Add a blank line between them to be clear that they are different.

In fact, from "view" that is a problem with each header and the syntax that follows it (no space between "Additional recommended arguments" and the arguments--they are on the same line). That might just be a github thing though